internal/cmd/config.go

```
@@ -48,12 +51,19 @@ func downloadFile(url string) (content string, err error) {
 	return b.String(), nil
 }
 
-func render(data string, foo any) string {
-	t := template.New("cloudConfig template").Delims("[[", "]]").Option("missingkey=zero")
-	t, _ = t.Parse(data)
```

:man_facepalming: we should stop hiding errors... Good catch!